### ISSUE TITLE
va-text-input: Make max length text more accessible

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#27940]
## Description

The `<va-text-input>` includes a `maxLength` property. When the max is reached, additional text is added below the input. To allow a screenreader to read out this dynamically added text, the text may need an alert role or `aria-describedby` attribute - the best solution should be determined by an accessibility expert.

<img width="598" alt="Screen Shot 2021-07-27 at 2 17 20 PM" src="https://user-images.githubusercontent.com/136959/127215168-c202d07a-3af7-460d-bb04-01a7da7bb9d7.png">

## Tasks

- [ ] Make max length text more accessible
- [ ] Accessibility specialist review

## Definition of done

- [ ] Tasks completed
- [ ] Accessibility review completed

**Comment by pdavies88** [comment]
@CourtneyChambers77  you can test via: https://60f9b557105290003b387cd5-iwpvfodnkz.chromatic.com/?path=/docs/components-va-text-input--default
Add a number to the maxlength field and then test with a screenreader to validate
![Screen Shot 2022-01-26 at 10.09.16 AM.png](https://images.zenhubusercontent.com/618570fbb420f53ff83732e5/932f11c0-ccb5-44e3-ae7a-4ac31c061751)

**Comment by CourtneyChambers77** [comment]
Able to select input field with keyboard and able to input text within the input element and tested using JAWS - Passes
